signature/mechanism_openpgp.go

```diff
@@ -147,7 +147,7 @@ func (m *openpgpSigningMechanism) Verify(unverifiedSignature []byte) (contents [
 		return nil, "", err
 	}
 	if md.SignatureError != nil {
-		return nil, "", fmt.Errorf("signature error: %v", md.SignatureError)
+		return nil, "", fmt.Errorf("signature error: %w", md.SignatureError)
```

Comment:
\*shrug\* this doesn’t _hurt_, but it’s also not _useful_: callers should not be assuming the `openpgp` build tag, so they should not be matching on the returned error.

Reply:
I agree that it's not an improvement, except maybe for code uniformity (so the rule "use %w for error wrapping" always applies).